### ISSUE TITLE
Can I add date time to note highlight template?

### ISSUE THREAD

**Issue by ivostork** [renehernandez/obsidian-readwise#65]
Hello thank you for you awesome plugin. Can I somehow add timestamp for the note highlight? Thank you very much!

**Comment by renehernandez** [comment]
Good idea! I'll add a PR with the addition of an `updated` field for highlights. It won't be on the default highlight template, but you would be able to add it to your custom highlight template
